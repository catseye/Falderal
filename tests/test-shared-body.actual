FalderalSyntaxError: line 19: test input must be preceded by test body
