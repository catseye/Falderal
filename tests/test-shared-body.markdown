Falderal test shared test bodies
--------------------------------

Test that tests with input sections can re-use the previous test's test body.

    -> Functionality "Silly Interpreter" is implemented by
    -> shell command "python silly-interpreter.py %(test-body-file)"

    -> Tests for functionality "Silly Interpreter"

    | print x
    = 

    | read x
    | print x
    + meow
    = meow

    + purr
    = purr

    | read x
    | read y
    | print y
    | print x
    + meow
    + purr
    = purr
    = meow

    + dog
    + cat
    = cat
    = dog
