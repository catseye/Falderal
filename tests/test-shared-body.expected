--------------------------------
Total test runs: 5, failures: 0
--------------------------------
